### ISSUE TITLE
Internal GID/UID overlaps with other group- a/o user-ids depending on used host system

### ISSUE THREAD

**Issue by starwarsfan** [SonarSource/docker-sonarqube#370]
The _group_ and _user_ to run SonarQube is created as a system account using this command:

`RUN groupadd -r sonarqube && useradd -r -g sonarqube sonarqube`

This leads to GID=999 and UID=999. So far, so good. But if you run the image lets say on CentOS with bind mounts to data directories on the host, you run into conflicts with existing GIDs/UIDs:

```
admin@sonartest { ~ }  » cat /etc/group | grep 999
ssh_keys:x:999:
admin@sonartest { ~ }  » cat /etc/passwd | grep 999
systemd-bus-proxy:x:999:997:systemd Bus Proxy:/:/sbin/nologin
```

So why is the account created as a system account? It would be great to use a normal user account with a higher ID value, which could easily be mapped to another account on the host system.

(Yes, I know Docker volumes but that's not an option for other reasons ;) )

**Comment by michal-duda-sonarsource** [comment]
Out of curiosity, what is the reason you can't use Docker volumes?

**Comment by snawaz** [comment]
> Out of curiosity, what is the reason you can't use Docker volumes?

Because I'm deploying it using Kubernetes, and using K8S volumes (PV and PVC) whose ownership/permissions need to be matched with the `sonarqube` components's user/group. I'd like to do this from K8S CRDs itself.

**Comment by michal-duda-sonarsource** [comment]
https://github.com/SonarSource/docker-sonarqube/pull/379 changes the UID and GID of `sonarqube` user to `1000` but also adds support for running the image as any user. It will be released in SonarQube 8.3.
